Fast Pair: align after sysbuild migration (#16022)
nrfconnect/sdk-nrf: samples/bluetooth/fast_pair/locator_tag/README.rst

This sample uses the :ref:`bt_fast_pair_readme` and its dependencies and is configured to meet the requirements of the Fast Pair standard together with its FMDN extension.
For details about integrating Fast Pair in the |NCS|, see :ref:`ug_bt_fast_pair`.

> This sample enables both the ``SB_CONFIG_BT_FAST_PAIR`` and :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig options to allow building either with or without Sysbuild.

Author: same here

> With this option enabled, the build system calls the :ref:`bt_fast_pair_provision_script` to automatically generate the hexadecimal file that contains Fast Pair Model ID and Anti Spoofing Private Key.

Reviewer: ```suggestion
With this option enabled, the build system calls the :ref:`bt_fast_pair_provision_script`, which automatically generates a hexadecimal file containing Fast Pair Model ID and Anti Spoofing Private Key.
```

> For more details about enabling Fast Pair for your application, see :ref:`ug_bt_fast_pair_prerequisite_ops_kconfig`.

Author: Shouldn't we write something along these lines? (Also in other places)

That's how we referred to sections in the integration guide. For example:

_If you want to support the battery information also in the DULT module, follow the instructions in the :ref:`ug_bt_fast_pair_gatt_service_fmdn_battery_dult` section._

```suggestion
For more details about enabling Fast Pair for your application, see the :ref:`ug_bt_fast_pair_prerequisite_ops_kconfig` section in the integration guide.
```

Reviewer: Good point, we can definitely align it
